authentication/account/worker/user_deactivation_worker_test.go

```diff
@@ -120,6 +141,41 @@ func (s *UserDeactivationWorkerTest) TestDeactivateUsers() {
 		err = l.Close()
 		require.NoError(s.T(), err)
 	})
+
+	s.Run("user not found in reg app", func() {
+		// given
+		ctx, _, _ := testtoken.ContextWithTokenAndRequestID(s.T())
+		userToDeactivate := s.Graph.CreateUser()
+		userToDeactivate.User().Cluster = "starter-us-east-2a"
+		err := s.Application.Users().Save(ctx, userToDeactivate.User())
+		require.NoError(s.T(), err)
+		identityToDeactivate := *userToDeactivate.Identity()
+		identityToDeactivate.LastActive = &ago40days
+		identityToDeactivate.DeactivationNotification = &ago30days
+		now := time.Now()
+		identityToDeactivate.DeactivationScheduled = &now
+		err = s.Application.Identities().Save(ctx, &identityToDeactivate)
+		require.NoError(s.T(), err)
+		mockRemoteCalls(userToDeactivate.User(), identityToDeactivate, s.Configuration, 404)
+		mockCheCalls(s.Configuration)
```

Comment:
or you could simply use a mock of the UserService and verify that its `DeactivateUser(...)` func was called ?

Reply:
Yup, but I started mocking few services and then continued ;-)